src/main/java/com/start/learn/human/Human.java

```diff
@@ -0,0 +1,60 @@
+package com.start.learn.human;
+
+import java.time.LocalDate;
+
+public class Human {
+    private String name;
+    private String surname;
+    private final LocalDate dayOfBirth;
+    private int humanId = assignId();
+    private static int nextHumanId = 1;
+
+    private static int assignId() {
+        int humanID = nextHumanId;
+        nextHumanId++;
+        return humanID;
+    }
+
+    public static int getNextHumanId() {
+        return nextHumanId;
+    }
+
+    public Human(int year, int month, int day) {
+
+        dayOfBirth = LocalDate.of(year, month, day);
+
+    }
+
+    public Human(String eName, String eSurname, int yearOfBirth, int monthOfBirth, int dayOfBirth) {
+        name = eName;
+        surname = eSurname;
+        this.dayOfBirth = LocalDate.of(yearOfBirth, monthOfBirth, dayOfBirth);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public LocalDate getDayOfBirth() {
+        return dayOfBirth;
+    }
+
+    public void setNewNameAndSurname(Human person) {
+        this.name = "Mister";
+        this.surname = "White";
+    }
+
+    public static void printHumanInfo(Human person) {
```

Comment:
toString() for @ Override function from Object class, so than any print result function that use object.toString() will use your implementation

Reply:
done